code-studio/package.json

```diff
@@ -18,27 +18,32 @@
     "lint-css": "sass-lint src/css/*css",
     "lint-js": "jshint src/js/*.js",
     "build": "npm run copy-assets && npm run build-js && npm run build-css",
+    "build:dist": "npm run lint && npm test && npm run clean && npm run copy-assets && npm run build-css && npm run build-js:min && npm run uglify-js",
```

Comment:
At what point do you think this gets too long/complex for us to want it here? I'm not sure of the answer, but certainly at some level of complexity having it as a one-liner becomes counter productive.